packages/shared/lib/services/sync/manager.service.ts

```diff
@@ -131,12 +132,14 @@ export class SyncManagerService {
                 }
 
                 const createdSync = await createSync(connection.id as number, syncName);
-                orchestrator.scheduleSyncHelper(
+
+                // Is this not awaited on purpose?
+                void orchestrator.scheduleSyncHelper(
                     connection,
                     createdSync as Sync,
                     syncConfig as ProviderConfig,
                     syncName,
-                    { ...sync, returns: sync.models, input: '' },
+                    { ...sync, returns: sync.models, input: '' } as NangoIntegrationData,
```

Comment:
let's find a solution to this sync_type incompatibility instead of casting. I am happy to pair if you want to